### PR TITLE
PD/SSFA loading data tests&improvements - experimental

### PR DESCRIPTION
**Pull request by adi130987**

### CODE REVIEW

**Comment by adi130987** [comment]
connects unicef/etools-issues#563

**Comment by adi130987** [comment]
depends on https://github.com/unicef-polymer/etools-profile-dropdown/pull/7

File: pmp/app-shell.html [hunk 1]
```diff
@@ -295,133 +290,81 @@
...
-          return;
-        }
-        if (this.route && this.baseUrl) {
-          var subroute = JSON.parse(JSON.stringify(this.route));
-          subroute.path = this.route.path.substring(subroute.path.indexOf(this.baseUrl) + this.baseUrl.length);
-          this.set('subroute', subroute);
-        }
-        this.set('page', routePage);
+        this.debounce('routePageChanges', function() {
+          if (!!!routePage) {
```
Comment (daniel-tabarcea) [inline pmp/app-shell.html]: Why the triple negation?

Comment (adi130987) [reply pmp/app-shell.html]: Old code, we can keep just one negation.

File: pmp/app-elements/interventions/behaviors/intervention-permissions-behavior.html [hunk 2]
```diff
@@ -259,8 +275,14 @@
         }
         this.setInterventionPermissions(false, this.originalIntervention.permissions);
```
Comment (daniel-tabarcea) [inline pmp/app-elements/interventions/behaviors/intervention-permissions-behavior.html]: originalIntervention should be added to the properties object

File: pmp/pages/page-interventions.html [hunk 3]
```diff
@@ -59,19 +56,6 @@
         active="{{newInterventionActive}}"
```
Comment (daniel-tabarcea) [inline pmp/pages/page-interventions.html]: newInterventionActive should be added to properties

File: pmp/pages/page-agreements.html [hunk 4]
```diff
@@ -91,67 +80,77 @@ <h1 main-title>
...
                     attr-for-selected="name"
-                    fallback-selection="list"
                     role="main">
 
-          <div class="page" name="list">
+          <template is="dom-if" if="[[_pageEquals(agreementPage, 'list')]]">
             <agreements-list id="list"
-                active="[[agreementListActive]]"
-                csv-download-url="{{csvDownloadUrl}}"
-                url-params="[[agreementQueryParams]]">
```
Comment (daniel-tabarcea) [inline pmp/pages/page-agreements.html]: agreementQueryParams should be added to properties

File: pmp/pages/page-partners.html [hunk 5]
```diff
@@ -105,45 +94,58 @@ <h1 main-title>
...
-          <div class="page" name="overview">
-            <partner-overview partner="[[partner]]"></partner-overview>
-          </div>
-          <div class="page" name="details">
+                           name="list"
+                           show-only-government-type="[[showOnlyGovernmentType]]"
+                           current-page="[[currentPage]]"
+                           active="[[listActive]]"
+                           csv-download-url="{{csvDownloadUrl}}"
+                           url-params="[[queryParams]]"></partners-list>
```
Comment (daniel-tabarcea) [inline pmp/pages/page-partners.html]: queryParams should be added to properties

**Pull request merged**